beat/Functions/posturographic_routine_perturbation.m

```diff
@@ -12,9 +12,9 @@
 
 platformdata=csv2cell(PlatformData, ";");
 platformdata_header=platformdata(1,:);
-cx=find(strcmpi(platformdata_header, 'cop_x'), 1); %% cop x component column
-cy=find(strcmpi(platformdata_header, 'cop_y'), 1); %% cop y component column
-cop=cell2mat(platformdata(2:end,cx:cy))*0.001; %converted in m
+cx=find(strcmpi(platformdata_header, 'CoP_x'), 1); %% cop x component column

```

Comment:
I don't how if the search is case sensitive, because the Platform data file of experiment 6 contains `cop_x`, and not  `CoP_c`. 

We will see if the test check goes well. If not this may need to be corrected

Reply:
So the CI passed, so it does not seem to be case sensitive.